status-im/status-react
# Add caching

fixes https://github.com/status-im/status-mobile/issues/20975

This commit integrates caching of the token balances.

To test:

1) Login when connection is working, make sure you have all the balances.
2) Logout, go offline
3) Login again

You should see the amount of tokens you have (unless it's less that 0.000001, that will be displayed as 0, but we could show it). 
Price in USD will not be shown as price is not cached yet.
